src/cryptoadvance/specter/wallet.py

```diff
@@ -363,6 +364,28 @@ def rescanutxo(self, explorer=None):
         t = threading.Thread(target=self._rescan_utxo_thread, args=(explorer,))
         t.start()
 
+    def export_labels(self):
+        labels = self.rpc.listlabels()
+        if "" in labels:
+            labels.remove("")
+        res = self.rpc.multi([
+            ("getaddressesbylabel", label)
+            for label in labels
+        ])
+        return [{ labels[i]: list(result['result'].keys()) } for i, result in enumerate(res)]
+    
+    def import_labels(self, labels):
+        # format: [
+        #   {'label': ['address', 'address2']}
+        # ]
+        rpc_calls = []
+        for label_row in labels:
+            label = list(label_row.keys())[0]
+            addresses = label_row[label]
+            for address in addresses:
+                rpc_calls.append(("setlabel", address, label))
+        self.rpc.multi(rpc_calls)
```

Comment:
makes sense only if there are any rpc calls (that might not be the case if imported wallet doesn't have any labels)
```suggestion
        if len(rpc_calls)>0:
            self.rpc.multi(rpc_calls)
```